src/components/Article.js

```diff
@@ -1,22 +1,67 @@
 import styles from "./article.module.scss";
+import { Link, useSearchParams } from "react-router-dom";
+import starImage from "images/star-svgrepo-com.svg";
+import inactiveStarImage from "../images/star-inactive.com.svg";
+import { useState } from "react";
 
-export default function Articles({ article }) {
+export default function Articles({ article, index, number }) {
   const date = article.publishedAt.slice(0, 10);
+  const [searchParams] = useSearchParams();
+  const [favorites, setFavorites] = useState(
```

Comment:
Rename this state to something that associated with what it is. Because now it's not clear from the name what the state for